HPXMLtoOpenStudio/resources/location.rb

```diff
@@ -95,4 +95,26 @@ def self.get_climate_zone_iecc(wmo)
 
     return
   end
+
+  def self.get_epw_path(hpxml, hpxml_path)
+    epw_path = hpxml.climate_and_risk_zones.weather_station_epw_filepath
+
+    if not File.exist? epw_path
+      test_epw_path = File.join(File.dirname(hpxml_path), epw_path)
+      epw_path = test_epw_path if File.exist? test_epw_path
+    end
+    if not File.exist? epw_path
+      test_epw_path = File.join(File.dirname(__FILE__), '..', 'weather', epw_path)
+      epw_path = test_epw_path if File.exist? test_epw_path
+    end
+    if not File.exist? epw_path
+      test_epw_path = File.join(File.dirname(__FILE__), '..', '..', 'weather', epw_path)
```

Comment:
If this is moving into a resources subdir, then you need to add another `'..'` to these locations.